endpoints/authentication/src/main.cpp

```diff
@@ -614,7 +567,7 @@ namespace irods::http::handler
 							.get<int>();
 					auto bearer_token = irods::http::process_stash::insert(authenticated_client_info{
 						.auth_scheme = authorization_scheme::openid_connect,
-						.username = std::move(irods_name),
+						.username = *irods_username,
```

Comment:
std::move?

Reply:
Done, check to see if you like the method, though.